fhir/resources/core/fhirabstractmodel.py

```diff
@@ -57,7 +57,7 @@ def json_dumps(v, *, default, option=0, return_bytes=False):
 FHIR_COMMENTS_FIELD_NAME = "fhir_comments"
 
 
-class WrongResourceType(PydanticValueError):
+class WrongResourceType:
     code = "wrong.resource_type"
     msg_template = "Wrong ResourceType: {error}"
```

Comment:
I think we ultimately want to remove this but I'm still unsure of the best way to replace it